docs: adds documentation for personal api tokens (#2164)
Unleash/unleash: website/docs/user_guide/personal-api-token.mdx

> ---

Reviewer: Can you move and rename this file to: `website/docs/how-to/how-to-create-personal-api-tokens.mdx`, please?

It's a how-to guide, so that's where it should go 🤓

Author: Definitely willing and able. Honestly this feels like good practise at getting more familiar with the docs which is something I want to do anyway

...but given how much it requires I'm also feeling separate PR

> id: personal-api-token

Reviewer: I suggest removing ids

```suggestion
```

title: How to create Personal API Tokens
---

:::info availability

Personal API tokens are planned to be released in **Unleash 4.17**.

:::

> Personal API tokens are a tool to enable a user to use the Admin API as themselves with their own set of permissions, rather than using an admin token. See [_how to use the Admin API_](../advanced/api_access) for more information.

Reviewer: I suggest linking to the reference docs. I'm also hesitant about linking to the api-access doc. I think it's a bit outdated. Do you think this would be sufficient?

```suggestion
[Personal API tokens](../reference/api-tokens-and-client-keys.mdx#personal-access-tokens) enable a user to use the Unleash admin API as themselves instead of as an global "admin" user.
```

Author: Yeah, I think that looks good, this is probably me getting too excited about adding links

## Step 1: Navigate to the personal API tokens page {#step-1}

Open the user profile pane in the admin UI and select the _view user profile_ menu item (available at the URL `/profile`).

![The admin UI navigation "user profile" menu with the view user profile menu item selected.](/img/create-pat-1.png)

Select the "Personal API Tokens" menu item.

![The user profile page with the "Personal API Tokens" menu item highlighted.](/img/create-pat-2.png)

> ## Step 2: Navigate to New Token {#step-2}
> 
> Navigate to "New Token".
> 
> ![The New Token element highlighted. ](/img/create-pat-3.png)
> 
> ## Step 3: Fill in the create personal API token form and create it {#step-3}
> 
> Give your token a description and optionally set an expiry date. By default the expiry date is set to 30 days.
> 
> ![The New Token form with the description text box and create element highlighted. ](/img/create-pat-4.png)
> 
> ## Step 4: Save your token {#step-4}

Reviewer: I think these two steps can be joined together.

```suggestion
## Step 2: Create a personal API token {#step-2}

Access the token creation form using the "New token" button.

![The New Token element highlighted. ](/img/create-pat-3.png)

Give your token a description and optionally set an expiry date. The default expiry date is set to 30 days from creation.

![The New Token form with the description text box and create element highlighted. ](/img/create-pat-4.png)

## Step 3: Save your token {#step-3}
```
